include/picongpu/plugins/PngPlugin.hpp

```diff
@@ -24,6 +24,9 @@
 
 #include "picongpu/particles/traits/SpeciesEligibleForSolver.hpp"
 #include "picongpu/plugins/ILightweightPlugin.hpp"
+#include "picongpu/plugins/PluginRegistry.hpp"
+#include "picongpu/plugins/output/images/PngCreator.hpp"
+#include "picongpu/plugins/output/images/Visualisation.hpp"
```

Comment:
We need it at the end of the files to register the PngPlugin.  IN the  past there was also the possibilty to send data ofer linux sockets instead of creating PnG's, therefore PngCreator is a template parameter.

Reply:
I see. Thanks for the explanation.